core/src/main/java/hudson/WebAppMain.java

```diff
@@ -385,7 +385,7 @@ public void contextDestroyed(ServletContextEvent event) {
             terminated = true;
             Thread t = initThread;
             if (t != null && t.isAlive()) {
-                LOGGER.log(Level.INFO, "Shutting down a Jenkins instance that was still starting up", new Throwable("reason"));
+                LOGGER.log(Level.FINEST, "Shutting down a Jenkins instance that was still starting up", new Throwable("reason"));
```

Comment:
Would rather go with upgrade over downgrade here, unless this would be a redundant message.